test/cli/cli-iexec-voucher.test.js

```diff
@@ -131,4 +129,113 @@ describe('iexec voucher', () => {
       );
     });
   });
+
+  describe('authorize', () => {
+    let requesterAddress;
+
+    beforeAll(async () => {
+      userWallet = await setRandomWallet();
+      voucherType = await createVoucherType(testChain)({});
+      await createVoucher(testChain)({
+        owner: userWallet.address,
+        voucherType,
+        value: 1000,
+      });
+      requesterAddress = await getRandomAddress();
+    });
```

Comment:
Do we need to run that again even if it's already done in the more global `beforeAll` line 34?